runtime/src/main/java/jp/co/soramitsu/runtime/multiNetwork/connection/ConnectionPool.kt

```diff
@@ -29,6 +33,36 @@ class ConnectionPool @Inject constructor(
     private val connectionWatcher = MutableLiveData<Unit>()
 
     init {
+        connectionWatcher.switchMap {
+            val connListFlow = pool.map {
+                it.value.isConnecting.map { isConnecting ->
+                    it.value.chain to isConnecting
+                }
+            }
+            val connChainsListFlow = combine(connListFlow) { chains ->
+                val issues = chains.filter { (_, isConnecting) -> isConnecting }.map { (chain, _) ->
+                    NetworkIssueItemState(
```

Comment:
If you don't need the isConnecting parameter here, you can filter by it while initializing the connListFlow variable.